src/AutorestSwift/View Models/RequestViewModel.swift

```diff
@@ -60,7 +60,7 @@ struct RequestViewModel {
 
         // Determine which kind of request body snippet to render
         if method == "patch" {
-            if let contentTypeParameter = request.parameter(for: "Content-Type"),
+            if let contentTypeParameter = request.parameter(for: "contentType"),
```

Comment:
So the definition of this header in AzureCore is `Content-Type`. This suggests to me that the chat Swagger is wrong and not autorest.

Reply:
In code model, the name is `contentType` and the serialized name is `Content-Type`. But in the request.parameter() method, we look for `parameter name` instead of `parameter serialized name` in the search.  The change to search in 'parameter name '  is introduced by a recent change. That is why it is broken.